docs/sections/user_guide/cli/drivers/esg_grid.rst

```diff
@@ -7,15 +7,15 @@ The ``uw`` mode for configuring and running the :ufs-utils:`regional_esg_grid<re
 
    $ uw esg_grid --help
     usage: uw esg_grid [-h] [--version] TASK ...
-    
+
```

Comment:
Under `docs/` remove trailing whitespace that often creeps in when pasting `--help` output from a terminal.